source/MaterialXView/Material.cpp

```diff
@@ -609,3 +682,36 @@ mx::ShaderPort* Material::findUniform(const std::string& path) const
     }
     return port;
 }
+
+void Material::persistUniformChanges()
```

Comment:
After some discussion with ILM, it would be better to directly update the Elements on the document at the time they are modified in the Property Editor. Thus there is no need to copy later on.